### PR TITLE
Adding blog post for zenbleed

### PR DESCRIPTION
**Pull request by bennyvasquez**

### CODE REVIEW

File: content/blog/zenbleed-patch-call-for-testing.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+title: "Testers needed: Zenbleed patch for AlmaLinux 8 and 9"
+type: blog
+author: 
+ name: "Andrew Lukoshko"
+ bio: "Release Engineering Lead"
+ image: /users/alukoshko.jpg
+date: '2023-07-24'
+post:
+    title: "Testers needed: Zenbleed patch for AlmaLinux 8 and 9"
+    image: /blog-images/23.07.24.zenbleed.png
```
Comment (jonathanspw) [inline content/blog/zenbleed-patch-call-for-testing.md:11]:
```suggestion
  title: "Testers needed: Zenbleed patch for AlmaLinux 8 and 9"
  image: /blog-images/23.07.24.zenbleed.png
```

Comment (bennyvasquez) [reply content/blog/zenbleed-patch-call-for-testing.md:11]: I'm hesitant to remove those spaces, as they've been in all of our other blog posts, and I know md can get picky about spaces.

File: content/blog/zenbleed-patch-call-for-testing.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+Due to the risks involved in these patches, these packages are not yet in production and need testing! If you are willing to help provide us feedback, and have access to a **bare metal AMD system**, you can manually install them by pulling them from the AlmaLinux Build System.  
+
+To install the new RPM on AlmaLinux 8: 
+
+> dnf update https://build.almalinux.org/pulp/content/builds/AlmaLinux-8-x86_64-7032-br/Packages/l/linux-firmware-20230404-114.git2e92a49f.el8_8.alma.noarch.rpm
+
+For AlmaLinux 9:
+
+> dnf update https://build.almalinux.org/pulp/content/builds/AlmaLinux-9-x86_64-7033-br/Packages/l/linux-firmware-20230310-134.el9_2.alma.noarch.rpm
```
Comment (codyro) [inline content/blog/zenbleed-patch-call-for-testing.md:40]: We should be consistent for all commands / snippets. I propose we change this and the one above accordingly. 

`dnf update https://build.almalinux.org/pulp/content/builds/AlmaLinux-9-x86_64-7033-br/Packages/l/linux-firmware-20230310-134.el9_2.alma.noarch.rpm`

File: content/blog/zenbleed-patch-call-for-testing.md (hunk 1 above)
Comment (jonathanspw) [reply content/blog/zenbleed-patch-call-for-testing.md:11]: It shouldn't matter either way I guess, I just like consistency :)

File: content/blog/zenbleed-patch-call-for-testing.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+You can see the diff of the changes on [git.almalinux.org](https://git.almalinux.org/rpms/linux-firmware/commit/b908df53b1c732aa18435a947f8e3c6a45ebc72c). 
+
+
+## How do I install the updated packages?
+
+Due to the risks involved in these patches, these packages are not yet in production and need testing! If you are willing to help provide us feedback, and have access to a **bare metal AMD system**, you can manually install them by pulling them from the AlmaLinux Build System.  
+
+To install the new RPM on AlmaLinux 8: 
+
+> dnf update https://build.almalinux.org/pulp/content/builds/AlmaLinux-8-x86_64-7032-br/Packages/l/linux-firmware-20230404-114.git2e92a49f.el8_8.alma.noarch.rpm
```
Comment (bennyvasquez) [inline content/blog/zenbleed-patch-call-for-testing.md:36]:
```suggestion
`dnf update https://build.almalinux.org/pulp/content/builds/AlmaLinux-8-x86_64-7032-br/Packages/l/linux-firmware-20230404-114.git2e92a49f.el8_8.alma.noarch.rpm`
```

File: content/blog/zenbleed-patch-call-for-testing.md (hunk 2 above)
Comment (bennyvasquez) [inline content/blog/zenbleed-patch-call-for-testing.md:40]:
```suggestion
`dnf update https://build.almalinux.org/pulp/content/builds/AlmaLinux-9-x86_64-7033-br/Packages/l/linux-firmware-20230310-134.el9_2.alma.noarch.rpm`
```

File: content/blog/zenbleed-patch-call-for-testing.md (hunk 1 above)
Comment (bennyvasquez) [reply content/blog/zenbleed-patch-call-for-testing.md:11]: Yeah, it's consistently got spaces, so we'll keep it for now. Those are meant to be the opengraph details that get shared... when we get that fixed. :D

**Review by codyro — APPROVED** [review]

**Pull request merged**